docs/manual/scala/guide/build/ScalaComponents.md

```diff
@@ -0,0 +1,96 @@
+
+[ClusterComponents](api/com/lightbend/lagom/scaladsl/cluster/ClusterComponents.html)
+
+ * registers the node to the Akka Cluster. Tha Akka Cluster is required by Pub-Sub and Persistent Entity support and you will rarely need to use it explicitly.
+
+[PubSubComponents](api/com/lightbend/lagom/scaladsl/pubsub/PubSubComponents.html)
+
+ * provides [[Publish-Subscribe|PubSub#Publish-Subscribe]]. This requires `ClusterComponents`.
+
+[CassandraPersistenceComponents](api/com/lightbend/lagom/scaladsl/persistence/cassandra/CassandraPersistenceComponents.html)
+
+ * provides both Read-Side and Write-Side components for Cassandra-backed CQRS. It provides `ReadSideCassandraPersistenceComponents`
+  and `WriteSideCassandraPersistenceComponents` which you might want to use in isolation. See [[PersistentEntityCassandra]] for more info.
+
+[JdbcPersistenceComponents](api/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceComponents.html)
+
+ * provides both Read-Side and Write-Side components for Cassandra-backed CQRS. It provides `ReadSideJdbcPersistenceComponents`
+  and `WriteSideJdbcPersistenceComponents` which you might want to use in isolation. See [[PersistentEntityRDBMS]].
+
+[LagomDevModeServiceLocatorComponents](api/com/lightbend/lagom/scaladsl/devmode/LagomDevModeServiceLocatorComponents.html)
+
+ * provides the dev mode service locator. This is meant to be used by Lagom Services and other applications such as Play Apps that want to interact with Lagom Services in Dev Mode. See the [scaladocs](api/com/lightbend/lagom/scaladsl/devmode/LagomDevModeServiceLocatorComponents.html) for more details.
+
+[LagomDevModeComponents](api/com/lightbend/lagom/scaladsl/devmode/LagomDevModeComponents.html)
+
+ * provides the dev mode service locator and registers the services with it in dev mode. See [[Wiring together a Lagom application|ServiceImplementation#Wiring-together-a-Lagom-application]].
+
+[StaticServiceLocatorComponents](api/com/lightbend/lagom/scaladsl/client/StaticServiceLocatorComponents.html)
+
+ * provides a Service Locator that always resolves the same URI for a given Service name.
+
+[ConfigurationServiceLocatorComponents](api/com/lightbend/lagom/scaladsl/client/ConfigurationServiceLocatorComponents.html)
+
+ * provides a Service Locator based on `application.conf` files. See [[deploying using static service locations|ProductionOverview#Deploying-using-static-service-locations]].
+
+[RoundRobinServiceLocatorComponents](api/com/lightbend/lagom/scaladsl/client/RoundRobinServiceLocatorComponents.html)
+
+ * provides a Service Locator that applies a Round Robin over the passed in sequence of URI.
+
+[CircuitBreakerComponents](api/com/lightbend/lagom/scaladsl/client/CircuitBreakerComponents.html)
+
+ * implementors of a Service Locator will need to extend this to reuse the Circuit Breaker config provided by Lagom.
```

Comment:
The section listing `StaticServiceLocatorComponents`, `ConfigurationServiceLocatorComponents`, `RoundRobinServiceLocatorComponents` and `CircuitBreakerComponents` could be extracted into a page of its own or merged into a single item of this, already long, list of `Components`.